src/Makefile.am

```diff
@@ -8,12 +8,6 @@ include $(srcdir)/src.mk
 
 noinst_HEADERS = $(SRC_H_FILES)
 
-if BUILD_TESTS
```

Comment:
Did you mean to remove this block?

Reply:
Yes, it was accidental duplication while editing.